arelle/plugin/validate/EDINET/PluginValidationDataExtension.py

```diff
@@ -502,7 +502,7 @@ def getFilingFormat(self, modelXbrl: ModelXbrl) ->  FilingFormat | None:
                 continue
             prefixes = {taxonomy.value for taxonomy in filingFormat.taxonomies}
             if not any(
-                str(fact.xValue).startswith(filingFormat.documentType.value) and
+                str(fact.xValue).strip().startswith(filingFormat.documentType.value) and
```

Comment:
Pre-existing, what `fact` variable is this using?  Is this using the last iteration of the `documentTitleFacts` loop above?  Is there a chance that `fact` can be `None` here?  Regardless, is there some way to make the logic clearer?

I was looking at the code because I was trying to understand what concept this code is verifying and trying to understand why `strip()` is needed rather than being able to rely on `collapse`.

Reply:
This statement iterates over `documentTitleFacts` which can not have `None` elements (our type-checking would have flagged that here).

I can simplify the `documentTitleFacts` population above to eliminate the conflicting local variable names.

I'm not sure what you mean re: relying on `collapse`.